docs/server-elasticsearch-setup.md

```diff
@@ -7,31 +7,28 @@ sidebar_label: Set up Elasticsearch
 ## Overview
 
 You can integrate [Elasticsearch](https://www.elastic.co/elasticsearch/) with the Temporal Server to enhance [Workflow search](/docs/server-workflow-search) functionality.
-Temporal Server versions < 1.6.0 require the use of Kafka + Zookeeper to use Elasticsearch.
-Temporal Server versions >= 1.6.0 do not require any additional dependencies to use Elasticsearch.
-The [Server versions and dependencies](/docs/server-versions-and-dependencies) page lists compatible versions of dependencies.
+The [Server versions and dependencies](/docs/server-versions-and-dependencies/#workflow-search) page describes which versions require the use of Kafka, and which ones do not.
 
 Integration with Elasticsearch is defined in the Server configuration files.
-For Server versions >= 1.6.0, all you need to do is edit the `persistence` section.
-For Server versions < 1.6.0, you will need to edit the `kafka` section as well.
+If you are not using Kafka, all you need to do is edit the `persistence` section.
+If you are using Kafka, you will need to edit the `kafka` section as well.
```

Comment:
Unfortunately, this is not true. `kafka` section can be omitted only in `v1.7.0`. It is required to be there for all current releases even if it is not used. There are two options here:
1. Release another patch for 1.5.x and 1.6.x and leave doc as is (less confusing but requires some work).
2. Fix doc and say that `kafka` section is required for versions `< 1.7.0`.

Reply:
We are not saying that it can be omitted from the config. We are saying that you don't need to edit that section.

If Kafka is disabled, per the instructions here: https://github.com/temporalio/temporal/releases/tag/v1.5.0 - then why would you need to edit this section? That doesn't make any sense to me - either the configuration is used or not?